profile-widget/divecartesianaxis.cpp

```diff
@@ -130,11 +131,22 @@ static double sensibleInterval(double inc, int decimals, bool is_time_axis)
 
 	double digits_factor = pow(10.0, digits);
 	int inc_int = std::max((int)ceil(inc / digits_factor), 1);
-	// Do "nice" increments of the leading digit. In general: 1, 2, 4, 5.
-	if (inc_int > 5)
-		inc_int = 10;
-	if (inc_int == 3)
-		inc_int = 4;
+	if (is_multiple_of_three)
+	{
+		// Do increments quantized to 3. In general: 1, 3, 6, 15
+		if (inc_int > 6)
+			inc_int = 15;
```

Comment:
Another small comment:

The jump to '15' is illogical.

You do accept '1' as the leading digit, so '10' should be fine as well. After all an increment equivalent to '10' could come from a different rounding above.

Reply:
This is intentional, cause otherwise you can get steps like 0, 10, 20, 30, 40 which is not multiple of 3. 
Having 15 will force steps 0, 15, 30, 45  and then 0, 30, 60, 120 for really deep dives.
And yes, I've tested that case.

Reply:
> This is intentional, cause otherwise you can get steps like 0, 10, 20, 30, 40 which is not multiple of 3.

Neither is 1 a multiple of 3. It appears illogical to me that 1, 2, 3, 4 is fine, but 10, 20, 30, 40 is not.

But my real point is: in the admittedly unlikely case, but stranger things have happened, that inc calculates as 10.0 straight, then you get `inc_int = 1` and `digits = 1` and thus increments 10, 20, 30, 40. No?

That said, since I will not be using this option I really don't mind. It just strikes me as weird code.

Reply:
I do agree, that it would be cleaner to also change the power of digit factor. And use 15 instead of 10 everywhere. 
```
-double digits_factor = pow(10.0, digits);
+double digits_factor = pow(15.0, digits);
```
But that would lead to bunch of if's.

Reply:
You know what. Maybe you right. Let me try it.

Reply:
Oh, wait. no. Mornings doesn't do good to me.
This is not going to work. It will turn whole numbering to septadecimal. 
NM